### DISCUSSION TITLE
Is there a file size limit?

### DISCUSSION THREAD

**Discussion by Gal-Peled** [jupyterlite/jupyterlite#858] [Q&A]
I get issue loading a file which is >1M and get an error 

![JL_Load issue](https://user-images.githubusercontent.com/91271356/198821934-5f28bb13-7023-4b94-9405-7471bdc85f1f.JPG)

**Comment by jtpio** [comment]
Thanks @Gal-Peled for reporting.

There is some logic to handle uploads in JupyterLab: https://github.com/jupyterlab/jupyterlab/blob/f68328f459f9e1c3c467d774fe1d488b320651ad/packages/filebrowser/src/model.ts#L396-L421

However the error message on the screenshot does mention any size limit.

On which instance are you trying to upload the notebook? (publicly available, self hosted...)

**Reply by jtpio** [reply]
On a side note JupyterLite does not seem to be setting `notebookVersion` as part of the page config. Which is something that could be done to make sure extensions using `PageConfig.getNotebookVersion()` get a proper value.

**Reply by Gal-Peled** [reply]
Hi
The file is a local file I have developed. Once I deleated some images from markdown it is not <1Mb and it runs with no issue.
